src/gwt/src/org/rstudio/studio/client/workbench/views/packages/PackagesPane.java

```diff
@@ -398,55 +431,113 @@ private boolean showButtonImpl(PackageInfo object)
       if (object.getPackageSource() == null)
          return false;
       
-      PackageInfo.Source source = PackageInfo.Source.valueOf(object.getPackageSource());
-      return source != PackageInfo.Source.Base;
+      String source = object.getPackageSource();
+      return source != "Base";
    }
    
    private void initPackagesTable()
    {
       packagesTable_.setKeyboardSelectionPolicy(KeyboardSelectionPolicy.DISABLED);
       packagesTable_.setSelectionModel(new NoSelectionModel<>());
-        
+      packagesTable_.addCellPreviewHandler(new CellPreviewEvent.Handler<PackageInfo>()
```

Comment:
We use this to handle behavior for the checkboxes for loading / unloading packages. This is done because those aren't _really_ checkboxes, they're buttons that load + unload packages, and the checkbox state should only update after a package has been successfully loaded / unloaded following a click.